### ISSUE TITLE
["Request"] Reduce tests log in build output

### ISSUE THREAD

**Issue by nomisRev** [arrow-kt/arrow#2150]
What would you like to see?

Currently we log `events "passed", "skipped", "failed", "standardOut", "standardError"`.

I think we should only log `skipped` or `failed`, since currently the logs are flooded with `PASSED` and it's hard to locate `FAILED` tests on the CI log.

**Comment by rachelcarmena** [comment]
Good idea @nomisRev !! Please, feel free to update https://github.com/arrow-kt/arrow/blob/master/subproject-conf.gradle to apply that improvement to all the Arrow libraries. Thank you so much!! :raised_hands:
